Library/Homebrew/cmd/tap.rb

```diff
@@ -60,7 +60,7 @@ def tap
         tap.fix_remote_configuration
       end
     elsif args.no_named?
-      puts Tap.select(&:installed?)
+      puts Tap.select(&:installed?).sort_by(&:to_s)
```

Comment:
```suggestion
      puts Tap.select(&:installed?).sort_by(&:name)
```

Reply:
It should be equivalent but I'll change it back for clarity.